_layouts/search-results.html

```diff
@@ -1,25 +0,0 @@
----
```

Comment:
Removed this search results page as the search.gov setup takes the user to an external site after searching.